template/src/components/ui/ConfirmationDialog.tsx

```diff
@@ -0,0 +1,67 @@
+import { Button, IconButton, styled } from "@mui/material";
+import * as React from "react";
+import { CustomDialog } from "./CustomDialog";
+import { Icon } from "./Icon";
+
+const HeaderContainer = styled("div")({
+    display: "flex",
+    justifyContent: "space-between",
+    alignItems: "center",
+});
+
+const MessageContainer = styled("div")({
+    textAlign: "left",
+});
+
+const ButtonContainer = styled("div")({
+    display: "flex",
+    alignItems: "center",
+    gap: 16,
+});
+
+const CancelLabel = styled("p")({
+    fontWeight: 600,
+    lineHeight: "16px",
+});
+
+export const ConfirmationDialog = ({
+    open,
+    title,
+    message,
+    submitLabel,
+    cancelLabel,
+    onSubmit,
+    onCancel,
+    hideCancel,
+}: {
+    open?: boolean;
+    title: string;
+    message: React.ReactNode;
+    submitLabel?: string;
+    cancelLabel?: string;
+    onSubmit: () => void;
+    onCancel: () => void;
+    hideCancel?: boolean;
+}) => {
+    return (
+        <CustomDialog open={open}>
+            <HeaderContainer>
+                <h1 style={{ fontSize: 32 }}>{title}</h1>
```

Comment:
Please add some spacing per default between the title and the icon button otherwise there is no space between them if the title is long.

Reply:
[add spacing to icon button](https://github.com/allaboutapps/cra-template-aaa-typescript/pull/23/commits/6cfe28e52e8482f4f19962cac555606af44ae78d)